tests/bfd/test_bfd.py

```diff
@@ -105,72 +142,104 @@ def stop_ptf_bfd(ptfhost):
-    assert bfd_state[0] == expected_state
-
+    if not SkipStateCheckAssert:
+        assert bfd_state[0] == expected_state
+    else:
+        print ("Expected State ", expected_state, "actual state", bfd_state[0])
 
-def create_bfd_sessions(ptfhost, duthost, local_addrs, neighbor_addrs):
+def create_bfd_sessions(ptfhost, duthost, local_addrs, neighbor_addrs, dut_init_first, scale_test = False):
     # Create a tempfile for BFD sessions
     bfd_file_dir = duthost.shell('mktemp')['stdout']
     bfd_config = []
-
+    dut_buffer = ""
+    ptf_buffer = ""
+    if scale_test:
+        # Force the PTF initialization to be first if runnign scale test. Doing so that we can send bathces fo 50 commadns to ptf
+        # and keep the code readable.
+        assert( dut_init_first == False )
+    
     for idx, neighbor_addr in enumerate(neighbor_addrs):
-        duthost.shell("sonic-db-cli APPL_DB hmset 'BFD_SESSION_TABLE:default:default:{}' local_addr {}".format(neighbor_addr, local_addrs[idx]))
+        dut_buffer += "sonic-db-cli APPL_DB hmset 'BFD_SESSION_TABLE:default:default:{}' local_addr {} ;".format(neighbor_addr, local_addrs[idx])
         bfd_config.append({
             "BFD_SESSION_TABLE:default:default:{}".format(neighbor_addr): {
                 "local_addr": local_addrs[idx]
             },
             "OP": "SET"
         })
-        ptfhost.shell("bfdd-control connect local {} remote {}".format(neighbor_addr, local_addrs[idx]))
+        ptf_buffer +="bfdd-control connect local {} remote {} ; ".format(neighbor_addr, local_addrs[idx])
+        if scale_test and idx%50 == 0:
+            ptfhost.shell(ptf_buffer)
+            ptf_buffer = ""
 
+    if not dut_init_first and ptf_buffer != "":
+        ptfhost.shell(ptf_buffer)
     # Copy json file to DUT
+    duthost.shell(dut_buffer)
     duthost.copy(content=json.dumps(bfd_config, indent=4), dest=bfd_file_dir, verbose=False)
 
     # Apply BFD sessions with swssconfig
     result = duthost.shell('docker exec -i swss swssconfig /dev/stdin < {}'.format(bfd_file_dir),
                            module_ignore_errors=True)
     if result['rc'] != 0:
         pytest.fail('Failed to apply BFD session configuration file: {}'.format(result['stderr']))
+    if dut_init_first :
+        ptfhost.shell(ptf_buffer)
 
 
 def remove_bfd_sessions(duthost, local_addrs, neighbor_addrs):
     # Create a tempfile for BFD sessions
     bfd_file_dir = duthost.shell('mktemp')['stdout']
     bfd_config = []
-
+    cmd_buffer = ""
     for idx, neighbor_addr in enumerate(neighbor_addrs):
-        duthost.shell("sonic-db-cli APPL_DB hmset 'BFD_SESSION_TABLE:default:default:{}' local_addr {}".format(neighbor_addr, local_addrs[idx]))
+        cmd_buffer += "sonic-db-cli APPL_DB hmset 'BFD_SESSION_TABLE:default:default:{}' local_addr {} ;".format(neighbor_addr, local_addrs[idx])
```

Comment:
The same as in `create_bfd_sessions` function above. Moreover, `hmset` is for setting and it wouldn't work for removing